lib/enemyModTables.js

```diff
@@ -41,6 +38,7 @@ module.exports = function($) {
     }
 
     // push the last one too
+    enemy._id = hash(enemy.enemyName + enemy.ememyModDropChance + enemy.mods.length)
```

Comment:
Just wondering: Why?

The point of this string is not to be human readable at all, in fact, the user won't directly interact with this in any way, the only purpose is to make the hash target more strict instead of just using "text" which is the cause of duplicate identifiers.

If this text were in any way supposed to be readable, I would immediately agree but that's not the point in this case, so why? 😃

Reply:
1. Less chance of misunderstanding this as a numerical operation, since aspects of it are numerical
2. Subjectively: it looks better
3. Across several projects we enforce the usage of templates over concatenation, so from this point of view: consistency.

Reply:
(I don't think that I'm consistent with that in this project either tho :D)